easypost/services/beta_carrier_metadata_service.py

```diff
@@ -18,9 +18,11 @@ def __init__(self, client):
         self._client = client
 
     def retrieve_carrier_metadata(
-        self, carriers: Optional[List[str]] = None, types: Optional[List[str]] = None
+        self,
+        carriers: Optional[List[str]] = None,
+        types: Optional[List[str]] = None,
     ) -> List[Dict[str, Any]]:
-        """Get carrier metadata for all carriers on the EasyPost platform."""
+        """Get CarrierMetadata for all carriers on the EasyPost platform."""
```

Comment:
nit: Is it brand-name `CarrierMetadata` or is it `metadata about carriers`?

Reply:
Yes? Heh it could be either. This ties into that greater convo. I can put it back if desired, or "metadata about carriers" is good too

Reply:
I made your suggested change